vm/vmgs/vmgs_format/src/lib.rs

```diff
@@ -16,6 +16,9 @@ use zerocopy::AsBytes;
 use zerocopy::FromBytes;
 use zerocopy::FromZeroes;
 
+/// The default capacity of a VMGS disk in bytes, 4MB.
+pub const VMGS_DEFAULT_CAPACITY: u64 = 0x400000;
```

Comment:
little weird that this constant is part of the `vmgs_format`, no? This is more of a implementation-defined constant?

Reply:
That distinction is somewhat hard to draw, since VMGS is an adhoc format, basically defined by what implementations do, and every implementation in existence uses 4MB. It seems like a shame to hard code this value separately in the 5 different places we make these things.

Reply:
Yeah, I hear ya. And I understand the temptation / logic here.

If we want to take the pragmatic route here, lets summarize the rationale behind having this constant here (i.e: its not part of the spec explicitly)